### PR TITLE
[WIP] -Fix: Maker

### PR DESCRIPTION
**Pull request by 0xpeluche**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/maker-rwa exports TVL: 
        
 
 
    ethereum                  1.47 B
    
    total                    1.47 B

**Comment by llamatester** [comment]
The adapter at projects/maker exports TVL: 
        
 
 
    ethereum                  4.75 B
    
    total                    4.75 B

File: projects/maker-rwa/index.js [hunk 1]
```diff
@@ -1,185 +1,79 @@
...
+  gem: "address:gem",
+  dog: "address:dog",
+}
 
-  ilks.forEach((_, i) => {
-    const token = auths[i].toLowerCase()
-    if (_) joins.push(token)
-    else failed.push(token)
+const getJoins = async (api) => {
+  const logs = (await getLogs2({ api, target: MCD_VAT, fromBlock: START_VAT_BLOCK, topics: [VAT_topic], skipCache: true, skipCacheRead: true })).map(log => {
```
Comment (g1nt0ki) [inline projects/maker-rwa/index.js:20]: need to remove  `skipCache: true, skipCacheRead: true` same for line 34

File: projects/maker/index.js [hunk 2]
```diff
@@ -1,206 +1,74 @@
...
-      throw new Error('Failed both gem and dog calls')
-    }
-  }
-
-  toa = toa.filter(i => i[0].toLowerCase() !== ADDRESSES.ethereum.SAI.toLowerCase())
-  const symbols = await api.multiCall({ abi: 'erc20:symbol', calls: toa.map(t => t[0]) })
-  const gUNIToa = toa.filter((_, i) => symbols[i] === 'G-UNI')
-  toa = toa.filter((_, i) => symbols[i] !== 'G-UNI' && !symbols[i].startsWith('RWA'))
+const getDogs = async (api) => {
+  const logs = (await getLogs2({ api, target: MCD_DOG, fromBlock: START_DOG_BLOCK, topics: [DOG_topic], skipCache: true, skipCacheRead: true })).map(log => {
```
Comment (g1nt0ki) [inline projects/maker/index.js:35]: same here

**Comment by llamatester** [comment]
The adapter at projects/maker-rwa exports TVL: 
        
 
 
    ethereum                  1.47 B
    
    total                    1.47 B

**Comment by llamatester** [comment]
The adapter at projects/maker exports TVL: 
        
 
 
    ethereum                  4.76 B
    
    total                    4.76 B

**Comment by llamatester** [comment]
The adapter at projects/maker-rwa exports TVL: 
        
 
 
    ethereum                  1.48 B
    
    total                    1.48 B

**Comment by llamatester** [comment]
The adapter at projects/maker exports TVL: 
        
 
 
    ethereum                  4.77 B
    
    total                    4.77 B

**Comment by llamatester** [comment]
The adapter at projects/maker exports TVL: 
        
 
 
    ethereum                  4.71 B
    
    total                    4.71 B

**Pull request merged**